ci/build39.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+# Special build for python3.9 until numpy puts its own wheels up
+
+sudo apt-get install build-essential gcc xvfb
+pip install --no-deps -U pip wheel setuptools
+pip install python-dateutil pytz pytest pytest-xdist hypothesis
+pip install cython --pre # https://github.com/cython/cython/issues/3395
+
+git clone https://github.com/numpy/numpy
+cd numpy
```

Comment:
can we use the wheel instead?
iirc the existing dev build was already changed

Reply:
I tried above - but there are no 3.9 wheels available yet on scipy-wheels-night or the old rackcdn url

Reply:
hmm ok

Reply:
Will update when available - tracking in https://github.com/pandas-dev/pandas/issues/33948